### DISCUSSION TITLE
Generálás JAVA-ból

### DISCUSSION THREAD

**Discussion by tothlp** [nav-gov-hu/Online-Invoice-XSLT#2] [General]
Üdvözletem!

Próbáltam a megadott parancsokkal parancssorból meghívni a html generálást, de a következő hibát kaptam:
`Failure evaluating XPath expression {HU} on command line`

Ezt követően próbáltam egy java-s projektben a linkelt leírások alapján megoldani a html és a pdf generálást is, azonban nem sikerül a transzformáció, a doctype-re, illetve lezáratlan meta tag-ekre hivatkozva. Van esetleg esély arra, hogy kapjunk egy minimális java példakódot egy happy-path működéshez?

Köszönöm!

**Comment by renced42** [comment]
Kedves @tothlp 

Mivel nem látom, hogy mit csinálsz, mi van feltelepítve/letöltve mire kapod a hibát, nem látom az inputot ezért nem igazán tudok segíteni.
A nyitó oldalon található leírások alapján én most simán legeneráltam egy html-t.
